manual/src/manual.tex

```diff
@@ -20,13 +21,16 @@
 % Package for code examples:
 \usepackage{listings}
 \usepackage{alltt}
-\usepackage{lmodern}% for supporting bold ttfamily in code examples
 \usepackage[normalem]{ulem}% for underlining errors in code examples
 \input{ifocamldoc}
 \ifocamldoc\else
 \usepackage{changepage}
 \fi
 \input{macros.tex}
+
+% No ligatures in typewriter font
+\DisableLigatures{encoding = T1, family = tt* }
```

Comment:
Hevea doesn't support `\DisableLigatures` (but renders correctly `--help`). It is probably better to define a noop `\DeclareLigatures` macro in `macros.hva` (around `\DeclareUnicodeCharacters`?).

Reply:
Right.  Done.